frontend/src/APIClients/mutations/StoryMutations.ts

```diff
@@ -199,4 +199,17 @@ export const SOFT_DELETE_STORY = gql`
     }
   }
 `;
+
 export type SoftDeleteStoryResponse = { ok: boolean };
+
+// export const PROCESS_STORY = gql`
```

Comment:
Added this thinking that the ticket involved frontend changes as well. I think it's a good idea to keep this for the next person who works on the frontend side of this ticket